.gitignore

```diff
@@ -1 +1,19 @@
-src/bin/*.dll
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Tbh, it seems like all these are only expected to appear within `src/`.  Rather than relying on subdirectory matching, why not put all this stuff in `src/.gitignore`?

Reply:
The `*.pdb` pattern is supposed to also exclude files from /GameData/TestLite/Plugins/ since this is where I made VS copy the build results. The other patterns would indeed only match files inside /src/. Do you still want me to move the rest into /src/? Not sure what to do about `pdb` since those can be found both inside and outside of /src/.

Reply:
Then add `*.pdb` to `.gitignore`, and put the rest of these patterns in `src/.gitignore`, please.